Formula/tensorflow.rb

```diff
@@ -0,0 +1,52 @@
+class Tensorflow < Formula
+  desc "Machine Intelligence Library"
+  homepage "https://tensorflow.org"
+  url "https://github.com/tensorflow/tensorflow.git",
+    :tag => "v0.7.1",
+    :revision => "028d0b46004c921acd48fdd0ec18128d79e18bf4",
+    :shallow => false
+
+  env :std
```

Comment:
After you abort the hung build with Ctrl-C, run `brew gist-logs tensorflow`. It'll collect the build logs that were generated behind the scenes and post them as a new gist on GitHub.